src/pages/MyLots/MyLots.tsx

```diff
@@ -3,46 +3,35 @@ import { useDispatch } from 'react-redux';
 import { Container } from '../../shared/styled/containers/Container';
 import { PageHeader } from '../../shared/styled/headers/PageHeader';
 import Lot from './Lot/Lot';
-import { getParts } from '../../store/reducers/parts/partsActions';
 import { useTypedSelector } from '../../hooks/useTypedSelector';
+import { getLots } from '../../store/reducers/lots/lotsActions';
 
-const lotsMockArr = [
-  {
-    id: 1,
-    name: 'Detail 1',
-    description: 'Low Priority',
-    fitCars: ['Spider'],
-  },
-  {
-    id: 2,
-    name: 'Detail 2',
-    description: 'High Priority',
-    fitCars: ['Stelvio', 'Giulia'],
-  },
-  {
-    id: 3,
-    name: 'Detail 3',
-    description: 'High Priority',
-    fitCars: ['Stelvio', 'Spider'],
-  },
-];
+export interface LotsInterface {
+  objectId: string;
```

Comment:
what is `objectId` ?

Reply:
@No-Max this id gas been got from source JSON
![image](https://user-images.githubusercontent.com/10351028/149295570-4f0a2cf3-7cfa-4eb3-967b-5e38ad496281.png)

Reply:
why not just an `id`?

Reply:
This is source  syntax. I can change it to id.
Upper case for first letter is good? I can change it too

Reply:
better use small letters for objects properties names